programmers/체육복.js

```diff
@@ -0,0 +1,16 @@
+function solution(n, lost, reserve) {
+    let realLost = lost.filter(i => reserve.indexOf(i) === -1).sort((a, b) => a - b);
+    let realReserve = reserve.filter(i => lost.indexOf(i) === -1).sort((a, b) => a - b);
+    const studentHave = n - realLost.length; 
+    const spare = [];  
+    for (let i = 0; i < realLost.length; i++) {
+        for (let j = 0; j < realReserve.length; j++) {
+            if (realReserve[j] + 1 === realLost[i] || realLost[i] === realReserve[j] - 1 || realLost[i] === realReserve[j]) {
```

Comment:
논리 연산이 여러개 사용될 때는
아래처럼 필히 구분시켜주는 것이 좋습니다.
뇌정지 옵니다.
```suggestion
            if (realReserve[j] + 1 === realLost[i] || 
                realLost[i] === realReserve[j] - 1 || 
                realLost[i] === realReserve[j]) {
```